wiki/core/markdown/__init__.py

```diff
@@ -22,7 +22,7 @@ def core_extensions(self):
 
     def get_markdown_extensions(self):
         kwargs = settings.MARKDOWN_KWARGS
-        extensions = kwargs.get('extensions', [])
+        extensions = list(kwargs.get('extensions', []))
```

Comment:
Not sure if I'm understanding the issue fully -- but shouldn't it be a `set()` rather than a list to avoid duplicate entries?